File: CS/Q140616/CustomObject.cs
using System.ComponentModel;
namespace DXSample {
    public class CustomObject {
        public CustomObject(string name, string password) {
            this.Name = name;
            this.password = password;
        }
        private string name;
        public string Name {
            get { return name; }
            set { name = value; }
        }
        private string password;
        [PasswordPropertyTextAttribute(true)]
        public string Password {
            get { return password; }
            set { password = value; }
        }
    }
}

File: VB/Q140616/CustomObject.vb
Imports Microsoft.VisualBasic
Imports System.ComponentModel
Namespace DXSample
	Public Class CustomObject
		Public Sub New(ByVal name As String, ByVal password As String)
			Me.Name = name
			Me.password_Renamed = password
		End Sub
		Private name_Renamed As String
		Public Property Name() As String
			Get
				Return name_Renamed
			End Get
			Set(ByVal value As String)
				name_Renamed = value
			End Set
		End Property
		Private password_Renamed As String
		<PasswordPropertyTextAttribute(True)> _
		Public Property Password() As String
			Get
				Return password_Renamed
			End Get
			Set(ByVal value As String)
				password_Renamed = value
			End Set
		End Property
	End Class
End Namespace